util/mcu_gen.py

```diff
@@ -219,7 +219,7 @@ def __init__(self, name, cell_name, pad_type, pad_mapping, index, pad_active, pa
     self.pad_mapping = pad_mapping
     self.pad_mux_list = pad_mux_list
 
-    if('low' in pad_active):
+    if pad_active == 'low':
```

Comment:
what if there are commas `,` in the hjson file?

Reply:
There shouldn't.  Not in a valid Hjson file at least.
https://hjson.github.io/syntax.html
> **Do not add commas** or comments as they would become part of the string.

Note that I fixed the affected Hjson file in this commit, so that should not be an issue in this particular case.
(But feel free to ditch the second commit of the PR and pull only the first one if you're not sure about that yet.)

Also see issue #552 which I created just now, immediately after submitting this PR, as I realized about that issue while dealing with this one.